### PR TITLE
Add additional steps for creating address space

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: documentation/modules/proc-create-address-space-cli.adoc [hunk 1]
```diff
@@ -8,6 +8,22 @@
 
 .Procedure
 
+ifeval::["{cmdcli}" == "oc"]
+. Login as messaging tenant
```
Comment (jenmalloy) [inline documentation/modules/proc-create-address-space-cli.adoc]: Log in as a messaging tenant:

File: documentation/modules/proc-create-address-space-cli.adoc [hunk 2]
```diff
@@ -8,6 +8,22 @@
...
++
+[source,yaml,options="nowrap"]
+----
+{cmdcli} login -u developer
+----
+
+. Create project for messaging application
```
Comment (jenmalloy) [inline documentation/modules/proc-create-address-space-cli.adoc]: Create the project for the messaging application:

**Comment by lulf** [comment]
@jenmalloy Please have another look

**Comment by jenmalloy** [comment]
lgtm!

**Pull request merged**